test/Main.js

```diff
@@ -4,6 +4,11 @@ const enviroment = (new jsdom.JSDOM('', { runScripts: "outside-only" }));
 global.window = enviroment.window;
 global.document	= enviroment.window.document;
 
+// a dirty snabbdom style module fix
```

Comment:
This condition will always be true. `jsdom` includes `requestAnimationFrame` if you enable the `pretendToBeVisual` [flag](https://github.com/jsdom/jsdom/commit/8a6894c34f14b9131d0fe4acadb71dae4f49daca). Then you can just copy the property like usual, e.g. `global.window.requestAnimationFrame = enviroment.window.requestAnimationFrame`.

Reply:
You don't need to copy it, then. I'm not sure we need to enable `pretendToBeVisual`, so I'll just remove condition check and force-push.